gravityflow/gravityflow
# Display queue/expired date-time in step column of status table

Adds sub-data to the step column of status table to display date/time values if they exist:
* The scheduled time for the step
* The expiration time for the step

![screenshot_2018-08-01_21-40-11](https://user-images.githubusercontent.com/4549984/43557641-98018538-95d3-11e8-8c78-95c24f2a0bfb.png)

***To Test***:
* Create a form/workflow with multiple steps
* Ensure they have a scheduled start, expiration, or both.
* Create an entry or multiple and move them through the various steps.

***To Remove/Hide***
Override the CSS class step_status_queue or step_status_expires which are set via the status.css.
